src/main/kotlin/org/rucca/cheese/task/TaskMembershipService.kt

```
@@ -262,6 +570,32 @@ class TaskMembershipService(
+                    } catch (e: NotFoundError) {
+                        return RealNameInfoRequiredError(memberId)
+                    }
+                }
+
+                TaskSubmitterType.TEAM -> {
+                    // For team participants, check if all team members have real name info
+                    val (teamMembers, _) = teamService.getTeamMembers(memberId)
+                    for (teamMember in teamMembers) {
+                        try {
+                            userRealNameService.getUserIdentity(teamMember.user.id)
+                        } catch (e: NotFoundError) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)